app/views/users/edit.html.haml

```diff
@@ -7,6 +7,9 @@
 = content_for :sidebar do
   = render "admin/shared/sidenav_users", sidenav_tab: "users"
 
+= content_for :tabnav do
+  = render :partial => 'admin/shared/tabnav_users', :locals => {:secondary_tab => 'details'}
```

Comment:
Picky thing (and I'm assuming this was copy/paste from somewhere else): we're trying to transition to the shorter style for `render`: `render "admin/shared/tabnav_users", secondary_tab:  "details"` Also, using double quotes and 1.9 hash syntax.